config-coral.ini

```diff
@@ -9,6 +9,8 @@ DashboardURL = http://0.0.0.0:8000
 ScreenshotsDirectory = /repo/data/processor/static/screenshots
 EnableSlackNotifications = no
 SlackChannel = lanthorn-notifications
+; OccupancyAlertsMinInterval time is measured in seconds (if timeout <= 0 then no occupancy alerts are triggered)
```

Comment:
I didn't find the implementation of `(if timeout <= 0 then no occupancy alerts are triggered)`.
I think that we should implement the validation or remove the comment.